src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -589,6 +590,12 @@ private DataAccessRequest updateDarWithDocumentContents(
       DataAccessRequest dar,
       InputStream uploadInputStream,
       FormDataContentDisposition fileDetail) throws IOException {
+    //when we move updateDarWithDocumentContents to the service tier, we should incorporate the code
+    //below into that method
+    if (dar.getDraft() != true) {
```

Comment:
Seems odd to compare against `true`
```suggestion
    if (!dar.getDraft()) {
```

Reply:
Waiting for 1520 to land and then can change.